K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -138,6 +143,53 @@ public static void AddAggregates(this AggregateDictionary aggregateDictionary, s
             }
         }
 
+        /// <summary>
+        /// Get extended stats for standard deviation aggregate from a given <see cref="DataRow"/>.
+        /// </summary>
+        /// <param name="columnName">The column name.</param>
+        /// <param name="row">The row to be parsed.</param>
+        /// <param name="logger">ILogger object for logging.</param>
+        /// <returns><see cref="ExtendedStatsAggregate"></returns>
+        private static ExtendedStatsAggregate GetExtendedStatsAggregate(string columnName, DataRow row, ILogger logger)
+        {
+            logger.LogTrace("Get extended stats for standard deviation aggregate for {}", columnName);
+
+            var extendedStatsAggregate = new ExtendedStatsAggregate();
+
+            var jObject = (JObject)row[columnName];
+
+            if (jObject == null)
```

Comment:
This actually can never happen. if the value of `row[columnName]` is null, it will return a DBNull object, and fail at the cast in 159

Reply:
Done